notebooks/official/tensorboard/tensorboard_vertex_ai_pipelines_integration.ipynb

```diff
@@ -921,14 +831,28 @@
       },
       "outputs": [],
       "source": [
-        "# Delete GCS bucket.\n",
-        "! gsutil -m rm -r {BUCKET_URI}\n",
-        "\n",
         "# Delete TensorBoard instance.\n",
         "! gcloud ai tensorboards delete {TENSORBOARD_RESOURCE_NAME}\n",
         "\n",
-        "# Delete custom job.\n",
-        "job.delete()"
+        "# Delete Vertex AI pipeline.\n",
+        "job.delete()\n",
+        "\n",
+        "# Delete the CustomJob using its display name.\n",
+        "jobs = aiplatform.CustomJob.list(filter='display_name=\"Trainer\"')\n",
+        "if jobs:\n",
+        "    job = jobs[0]\n",
+        "    job.delete()\n",
+        "    print(\"Deleted CustomJob:\", job.display_name)\n",
+        "else:\n",
+        "    print(\"No CustomJob found with the display name 'Trainer'\")\n",
```

Comment:
backticks for `CustomJob `??

Reply:
It's inside print statement. Removed the print statements.